templates/shared_services/firewall/terraform/rules.tf

```diff
@@ -215,8 +15,8 @@ resource "azurerm_firewall_policy_rule_collection_group" "dynamic_network" {
         for_each = network_rule_collection.value.rules
 
         content {
-          name = rule.value.name
-          # description      = rule.value.description
+          name             = rule.value.name
+          description      = rule.value.description
```

Comment:
Is it confirmed that network rules now support a description?

Reply:
Will check. It deploys and works... can't remember why changed this.

Reply:
Good catch, no idea where this came from.